scripts/profiler/rtt_nordic_profiler_host.py

```diff
@@ -271,23 +272,23 @@ def _read_single_event_rtt(self):
             byteorder=self.config['byteorder'],
             signed=False)
         et = self.received_events.registered_events_types[id]
+        # Calculating timestamp
+        buf = self._read_bytes(1)
+        encoded = int.from_bytes(buf, byteorder=self.config['byteorder'],
+                                    signed=False)
+        decoded = 0
+        counter = 0
+        # Decoding using Variable-length quantity (VLQ)
```

Comment:
Btw. `The VLQ octets are arranged most significant first in a stream`.
I think we could leave it as it is, but not call it VLQ if it does not match encoding description :)